qa/README.md

```diff
@@ -5,6 +5,16 @@ Every pull request to the bitcoin repository is built and run through
 the regression test suite. You can also run all or only individual
 tests locally.
 
+Test dependencies
+=================
+Before running the tests, the following must be installed.
+
+Unix
```

Comment:
Just updated with a clarification that the instructions are for Ubuntu and Debian, as done in /doc/build-unix.md